### ISSUE TITLE
Installing holoviz on MacOS with Python 3.9.2 fails (long log included)

### ISSUE THREAD

**Issue by HWiese1980** [holoviz/holoviz#280]
#### ALL software version info
* Python 3.9.2
* MacOS Big Sur 11.2.3
* iTerm 2 3.4.4
* zsh 5.8 (x86_64-apple-darwin19.3.0)
* Apple clang version 12.0.0 (clang-1200.0.32.29)
* pip 21.0.1
* virtualenv 20.4.3
* asdf (used to install Python 3.9) v0.8.0-c6145d0

#### Description of expected behavior and the observed behavior
Installation runs through without errors

#### Complete, minimal, self-contained example code that reproduces the issue

```zsh
# install and activate python 3.9.2 via asdf
# install virtualenv via pip
# create and activate venv

pip install holoviz

# see lots of errors
```

#### Stack traceback and/or browser JavaScript console output
Here's a gist of the error log: https://gist.github.com/HWiese1980/6c1f37b9d94536de80f5f50b59dd0bb2

**Comment by HWiese1980** [comment]
Can anybody at least confirm that this is reproducible? Or am I alone with that error?

**Comment by jbednar** [comment]
HoloViz has a lot of dependencies, and I wouldn't expect that they are all available for Python 3.9 yet.  It's not something we've tested, but it still seems early to be testing it. We'll support 3.9 at some point, but no timeline for that has been set yet. Sorry for not responding earlier!

**Comment by HWiese1980** [comment]
> Sorry for not responding earlier!

No worries, I've of course got another environment with Python 3.8 at hand. I'd just love to use the new features of Python 3.9 as soon as possible, and I furthermore simply wanted to let you know about my experience with HoloViz and Python 3.9.

Anyway, Python 3.9 is already about half a year old, we're at v3.9.4 now. Just saying, but it's a shame (in general, not regarding Holoviz in particular) that there are still dependencies around that are not updated for 3.9 yet. Myy two cents...
